engines/gob/console.cpp

```diff
@@ -208,15 +208,18 @@ bool GobConsole::cmd_cheat(int argc, const char **argv) {
 bool GobConsole::cmd_listArchives(int argc, const char **argv) {
 	Common::Array<ArchiveInfo> info;
 
-	_vm->_dataIO->getArchiveInfo(info);
-
-	debugPrintf("   Archive    | Base | FileCount\n");
-	debugPrintf("--------------------------------\n");
-	for (Common::Array<ArchiveInfo>::const_iterator it = info.begin(); it != info.end(); ++it)
-		if (!it->name.empty())
-		debugPrintf("%13s |   %d  | %d\n", it->name.c_str(), it->base, it->fileCount);
-
-	return true;
+    _vm->_dataIO->getArchiveInfo(info);
+
+    debugPrintf(" Archive        | Base | FileCount | FileSize\n");
+    debugPrintf("----------------------------------------------\n");
+    for (Common::Array<ArchiveInfo>::const_iterator it = info.begin(); it != info.end(); ++it) {
+        if (!it->name.empty()) {
+            uint32 size = _vm->_dataIO->fileSize(it->name.c_str());
```

Comment:
What is the point of adding this file size? Is it different from the one we find in our browsers?

Reply:
To show the filesizes of the archives in KB, For all Archives which are used currently by the Engine to show their filesize, but maybe we can turn it into someting else? like in the original debug menus which show the read access in ms?

Reply:
This command is used for debugging purposes, to show a list of game archives, together with the number of files that they contain. Although the size of each archive can be added to the debug output, it doesn't seem to be related with the reason that this command was created (to list the number of files that are contained in each archive)